pkg/transforms/conversion.go

```diff
@@ -64,6 +65,7 @@ func (f Conversion) TransformToJSON(edgexcontext *appcontext.Context, params ...
 			// LoggingClient.Error(fmt.Sprintf("Error parsing JSON. Error: %s", err.Error()))
 			return false, errors.New("Error marshalling JSON")
 		}
+		edgexcontext.ResponseContentType = "application/json"
```

Comment:
You use `clients.ContentTypeJSON` in the test, why not use it here? Perhaps a constant should be added for XML as well?

Reply:
Yes, since the struct is public, I don't see a reason to use a setter method when it can be set simply by assigning a value to the struct property value.

I updated the code to use the `clients.ContentTypeJSON` for `TransformToJSON()`. Since there isn't a constant for XML, and to mirror the use of a constant, I created a new constant at the top of conversion.go called `const ContentTypeXML = "application/xml"`. I also updated `TransformToXML()` to use `ContentTypeXML`.